tests/src/main/java/net/neoforged/neoforge/debug/capabilities/VanillaHandlersTests.java

```diff
@@ -1,149 +0,0 @@
-/*
- * Copyright (c) NeoForged and contributors
- * SPDX-License-Identifier: LGPL-2.1-only
- */
-
-package net.neoforged.neoforge.debug.capabilities;
-
-import static net.neoforged.neoforge.fluids.capability.IFluidHandler.FluidAction.EXECUTE;
-import static net.neoforged.neoforge.fluids.capability.IFluidHandler.FluidAction.SIMULATE;
-
-import net.minecraft.core.BlockPos;
-import net.minecraft.core.Direction;
-import net.minecraft.network.chat.Component;
-import net.minecraft.world.level.block.Blocks;
-import net.minecraft.world.level.block.LayeredCauldronBlock;
-import net.minecraft.world.level.material.Fluids;
-import net.neoforged.neoforge.capabilities.BlockCapabilityCache;
-import net.neoforged.neoforge.capabilities.Capabilities;
-import net.neoforged.neoforge.fluids.FluidStack;
-import net.neoforged.testframework.annotation.ForEachTest;
-import net.neoforged.testframework.annotation.TestHolder;
-import net.neoforged.testframework.gametest.EmptyTemplate;
-import net.neoforged.testframework.gametest.ExtendedGameTestHelper;
-import net.neoforged.testframework.gametest.GameTest;
-import org.apache.commons.lang3.mutable.MutableInt;
-
-@ForEachTest(groups = "capabilities.vanillahandlers")
-public class VanillaHandlersTests {
```

Comment:
Don't remove these old tests. Keep them, and just wrap the new handler as an old one.